### PR TITLE
Remove unused bash from anago

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
Some code of anago appears unused, so we remove it.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by saschagrunert** [comment]
Manual testing:
- :heavy_check_mark: Stage: https://console.cloud.google.com/cloud-build/builds/1ff9f982-3491-4eeb-8eee-1dc720b108e8?project=kubernetes-release-test
- :heavy_check_mark: Release: https://console.cloud.google.com/cloud-build/builds/3cbf5fcd-e273-4140-8f33-b8a544a3afc9?project=kubernetes-release-test

File: anago [hunk 1]
```diff
@@ -230,76 +212,6 @@ copy_logs_to_workdir () {
   logrun -s cp -f $logfiles $WORKDIR
 }
 
-###############################################################################
-# Ensures we have write access to a specified registry
-# @param registry - A registry to check the ACLs for
-#
-ensure_registry_acls () {
```
Comment (xmudrii) [inline anago:237]: This function has one commented reference: https://github.com/kubernetes/release/blob/313f232977d63876d856647a0f9fd10aeb11bd59/anago#L260

Should we also remove that reference as we're removing the function?

Comment (saschagrunert) [reply anago:237]: Yep, removed that line as well as the comment above :+1:

**Review by xmudrii — APPROVED** [review]
/lgtm

**Pull request merged**